src/pages/SearchDetails/index.tsx

```diff
@@ -0,0 +1,31 @@
+import { useFetchShops } from 'api/search';
+import { useParams } from 'react-router-dom';
+import styles from 'pages/SearchDetails/SearchDetails.module.scss';
+import NavigationBar from 'pages/Search/components/NavigationBar';
+import LoadingView from './components/LoadingView';
+import SearchItem from './components/SearchItems';
+
+interface Props {
+  address: string,
+  placeName: string,
+  shopId: number,
+}
+
+export default function SearchDetails() {
+  const { searchQuery } = useParams();
+  const { isFetching, data: shops } = useFetchShops(searchQuery as string);
```

Comment:
`string | undefined` 타입을 `string`으로 강제하는 것보다, Non-null assertion operator(말이 길지 그냥 느낌표)를 통해 nullish 하지 않다고 단언할 수 있습니다.
```suggestion
  const { isFetching, data: shops } = useFetchShops(searchQuery!);
```